Formula/stlink.rb

```diff
@@ -15,15 +15,13 @@ class Stlink < Formula
   end
 
   depends_on "libusb"
+  depends_on "gtk+3" => :optional
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  depends_on "cmake" => :build
```

Comment:
I get far more bothered by `:build` dependencies being below everything else than I should do 😆